apps/chat/src/components/Folder/Folder.tsx

```diff
@@ -1110,9 +1110,9 @@ const Folder = <T extends ConversationInfo | PromptInfo | DialFile>({
           entity={currentFolder}
           entities={
             featureType === FeatureType.Chat
-              ? (allChildItems as ConversationInfo[]).filter(
-                  (item) => !item.isReplay,
-                )
+              ? (
+                  allChildItems as (Partial<Conversation> & ConversationInfo)[]
+                ).filter((item) => !item.isReplay && item.messages?.length)
```

Comment:
need to allow Playback
```suggestion
                ).filter((item) => item.isPlayback || (!item.isReplay && item.messages?.length))
```